### DISCUSSION TITLE
Effects Help

### DISCUSSION THREAD

**Discussion by jbreske** [forkineye/ESPixelStick#674] [Q&A] [answered]
Is there a manual or discussion for the effects settings? I'm not sure what the "effects flash settings" in Device Setup really does. Is there a way to set two color settings like purple and orange.

**Comment by MartinMueller2003** [comment, marked as answer]
The readme describes some of the functionality. The effects flash functionality was just added recently to simulate lightning while other effects are playing.

**Reply by jbreske** [reply]
Ah. That makes sense with the options that were there. Thanks.
